runtime/src/accounts_db.rs

```diff
@@ -4312,15 +4312,16 @@ impl AccountsDb {
     /// returns the pubkeys that are in 'accounts' that are already in 'existing_ancient_pubkeys'
     /// Also updated 'existing_ancient_pubkeys' to include all pubkeys in 'accounts' since they will soon be written into the ancient slot.
     fn get_keys_to_unref_ancient<'a>(
-        accounts: &'a [(&Pubkey, &StoredAccountMeta<'_>, u64)],
+        accounts: &'a [(&StoredAccountMeta<'_>, u64)],
```

Comment:
nit: I probably missed this in a previous review. It looks like u64 should be Slot.